NEWS.md

```diff
@@ -1,3 +1,22 @@
+# New in snapd 2.61:
+* Fix control of activated services in 'snap start' and 'snap stop'
+* Correctly reflect activated services in 'snap services'
+* Disabled services are no longer enabled again when snap is refreshed
+* interfaces/builtin: added support for Token2 U2F keys
+* interfaces/u2f-devices: add Swissbit iShield Key
+* interfaces/builtin: update gpio apparmor to match pattern that contains multiple subdirectories under /sys/devices/platform
+* interfaces: add a polkit-agent interface
+* interfaces: add pcscd interface
+* Kernel command-line can now be edited in the gadget.yaml
+* Only track validation-sets in run-mode, fixes validation-set issues on first boot.
+* Added support for using store.access to disable access to snap store
+* Support for fat16 partition in gadget
+* Pre-seed authority delegation is now possible
+* Support new system-user name  _daemon_
+* Several bug fixes and improvements around remodelling
+* Offline remodelling support
+* Offline device mode support
```

Comment:
what is this last line about?

is this the same as "Added support for using store.access to disable access to snap store" above?

Reply:
is it about device-service.access=offline? in which case it is too cryptic I think

Reply:
Good catch, can see we went a bit overboard with the NEWS added. I'll get that fixed

Reply:
It was not about the device-service.access (i believe I see different entries for that one in the commits), the last one was added from memory, and the first one was added from the changelog we generated, so we should have removed the last one